window-backend/app.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const app = express()
+
+app.use(express.json());
+
+app.get('/employee', function(req,res){
+    employee = [
```

Comment:
why are we not getting this from the db?

Reply:
Realised my commit was a bit wrong but this is just connecting the frontend to the backend and @roisingoodman1 was working on the db connection which should be coming together. Will amend commit message

Reply:
ok thats fine, what is the task number that @roisingoodman1 is working on? leave a comment to reflect this.

Reply:
https://github.com/MetalGearFridge/fuinneog/tree/task/sqlConnection link to branch - should be under same ticket - updated description to include this now

Reply:
editing branch name now to inc the ticket number